src/reducers/mazeReducer/mazeReducer.ts

```diff
@@ -60,6 +62,18 @@ const changeSpaceType = (
   return newMaze;
 };
 
+const updateSpaceProp = (
+  coord: Coord,
+  mazeInfo: MazeInfo,
+  prop: "path" | "visited"
```

Comment:
It might make sense to make an enum for `SpaceProps`

Reply:
That's smart

Reply:
Definitely not necessary for this PR, what you have here works perfect. But maybe down the line, or just like the next time you are working on those files